chains/substrate/executor/executor.go

```diff
@@ -115,16 +128,18 @@ func (e *Executor) Execute(msgs []*message.Message) error {
 	signing, err := signing.NewSigning(
 		msg,
 		e.sessionID(propHash),
+		span.SpanContext().TraceID().String(),
```

Comment:
Would probably send context here and in the context add traceID instead of directly depending on sending it.

Reply:
Same for all tss processes.

Reply:
yeah, removed it, actually do not need it there with closer look